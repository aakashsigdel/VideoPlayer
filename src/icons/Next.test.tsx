import * as React from 'react'
import { create } from 'react-test-renderer'

import { Next } from './Next'

describe('<Next />', () => {
  it('should render correctly', () => {
    const tree = create(<Next />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
