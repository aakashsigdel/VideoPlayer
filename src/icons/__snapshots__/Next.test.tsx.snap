// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Next /> should render correctly 1`] = `
<svg
  fill="#fff"
  height="24"
  viewBox="0 0 24 24"
  width="24"
>
  <path
    d="M6 18l8.5-6L6 6v12zM16 6v12h2V6h-2z"
  />
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
</svg>
`;
